### DISCUSSION TITLE
Support non-romanised languages like Chinese, Japanese, Korean, etc.

### DISCUSSION THREAD

**Discussion by pingshunhuangalex** [monkeytypegame/monkeytype#954] [Ideas] [answered]
Any plans to support non-romanised languages like Chinese, Japanese, Korean, etc.?

I guessed there may be some technical difficulties around it cuz you can't just listen to keystrokes anymore. Thanks.

**Comment by Miodec** [comment, marked as answer]
I mean yeah its planned, but needs a input rewrite, and it just seems a bit of a big task currently.
